report_xml/controllers/main.py

```diff
@@ -21,7 +24,17 @@ def report_routes(self, reportname, docids=None, converter=None, **data):
             response.data = response.data.strip()
             response.headers.set("Content-Type", "text/xml")
             response.headers.set('Content-length', len(response.data))
+            # set filename
+            action_report_obj = request.env['ir.actions.report']
+            report = action_report_obj._get_report_from_name(reportname)
+            filename = report.name
+            if docids:
+                ids = [int(x) for x in docids.split(",")]
+                obj = request.env[report.model].browse(ids)
```

Comment:
The name of this variable can be better no? :/ records? obj_rs?

Reply:
I agree, I have named it records